test/units/TEST-30-ONCLOCKCHANGE.sh

```diff
@@ -20,7 +26,14 @@ timedatectl set-timezone Europe/Kyiv
 
 while test ! -f /tmp/timezone-changed ; do sleep .5 ; done
 
-timedatectl set-time 2018-1-1
+# Get current time and add a future offset (e.g., 1 year and 1 month ahead)
+# Using date to calculate the future timestamp
```

Comment:
Here also. the comment is trivial, and duplicated. I think not necessary.

Reply:
done